doc/Security.xml

```diff
@@ -4867,6 +4875,55 @@ CSeq: 2
 User-Agent: ./onvifClient
 Accept: application/sdp]]></programlisting>
   </appendix>
+
+  <appendix xml:id= "b14_55m_51d">
+    <title>Cipher Reference</title>
+
+	<para>This Annex is <b>NOT</b> meant to be an exhaustive and definitive Cipher list that ONVIF is advocating as part of the ONVIF standard that readers of this specification are to use but just a guide, to a small subset of ciphers covering TLS 1.2 / 1.3 that are common recommendations issued as guidance by the bodies Cloudflare, IETF (TLS 1.2, TLS1.3), Mozilla, and ciphersuite.info (TLS 1.2, TLS 1.3).</para>
```

Comment:
Prefer a positive statement. Suggest to shorten to:
"This Annex is advocating a small subset of of ciphers ... "

Reply:
@HansBusch I agree and reworded Paragraph. Updated as requested.